pages/chat_page.py

```diff
@@ -0,0 +1,81 @@
+from .base_page import Page
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+class ChatPage(Page):
+
+    url = 'chat/'
+
+    def open_chat(self):
+        self.find_chat().click()
+
+    def find_chat(self):
+        self.context.wait.until(EC.presence_of_element_located((By.ID, 'status_dropdown')))
+        return self.context.driver.find_element_by_xpath("//span[text()='Alias room']")
+
+    def open_alias_menu(self):
+        self.find_plus_butt().click()
+
+    def find_plus_butt(self):
+        self.context.wait.until(EC.presence_of_element_located((By.ID, 'input_actions_dropdown-trigger')))
+        return self.context.driver.find_element_by_id("input_actions_dropdown-trigger")
+
+    def open_menu(self):
+        self.find_butt_in_dropdown_menu().click()
+
```

Comment:
please "inspect code" in pycharm and follow PEP8 style (empty lines between code blocks in this case)

Reply:
Done